services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None,
```

Comment:
If argument can be `None`, you should show it in type hint. Fix everywhere
```suggestion
    title: str | None = None,
```